docs/usage-cli.rst

```diff
@@ -0,0 +1,90 @@
+.. _usage-cli:
+
+Command-line Interface
+======================
+
+Usage
+-----
+
+::
+
+   $ dredd '<api-description-document>' '<api-location>' [OPTIONS]
+
+Example:
+
+::
+
+   $ dredd ./apiary.md http://127.0.0.1:3000
+
+Arguments
+---------
+
+.. _api-description-document-string:
+
+.. option:: api-description-document
+
+   URL or path to the API description document (API Blueprint, Swagger).
+   **Sample values:** ``./api-blueprint.apib``, ``./swagger.yml``, ``./swagger.json``, ``http://example.com/api-blueprint.apib``
+
+.. _api-location-string:
+
+.. option:: api-location
+
+   URL, the root address of your API.
+   **Sample values:** ``http://127.0.0.1:3000``, ``http://api.example.com``
+
+Configuration File
+------------------
+
+If you use Dredd repeatedly within a single project, the preferred way to run it is to first persist your configuration in a ``dredd.yml`` file. With the file in place you can then run Dredd every time simply just by:
+
+::
+
+   $ dredd
+
+Dredd offers interactive wizard to setup your ``dredd.yml`` file:
+
+::
+
+   $ dredd init
+
+See below how sample configuration file could look like. The structure is the same as of the :ref:`Dredd Class configuration object <configuration-object-for-dredd-class>`.
+
+.. code-block:: yaml
+
+   reporter: apiary
+   custom:
+     - "apiaryApiKey:yourSecretApiaryAPiKey"
+     - "apiaryApiName:apiName"
+   dry-run: null
+   hookfiles: "dreddhooks.js"
+   sandbox: false
+   server: rails server
+   server-wait: 3
+   init: false
+   custom: {}
```

Comment:
```suggestion
```
There is already a custom key declared, when I copy pasted this config my IDE went red, mentioning this here so I can close #1120